power-of-2/power-of-2.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	fmt.Println("The program determines if a number is a power of 2.")
+	fmt.Println("Enter the number:")
+	var i int //8; -3; 256; 377
+	fmt.Scanln(&i)
+	var g, h int
+	if i == 1 {
```

Comment:
optional: Can you change your program to avoid handling these cases using separate `if`s? Can you handle them in a loop?